setup.py

```diff
@@ -14,6 +14,7 @@
 
 install_requires = [
     'draftjs_exporter==0.7.0',
+    'jinja2>=2.9.5,<3'  # TODO: remove when it's not necessary (https://github.com/springload/wagtaildraftail/issues/13)
```

Comment:
@dnsl48 found that if we create a template node which inherit from [RichText](https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailcore/rich_text.py#L186) and provide our own methods, the [rich_text](https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailcore/templatetags/wagtailcore_tags.py#L54) filter will transparently consume Draft JSON data which means we can remove the need for template tags in this project (which also solve what was express in #13).